### PR TITLE
lstopo: add Spanish translation

### PR DESCRIPTION
**Pull request by kant**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [ ] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages.es/common/lstopo.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# lstopo
+
+> Muestra la topología de hardware del sistema.
+> Más información: <https://manned.org/lstopo>.
+
+- Muestra la topología resumida del sistema en una ventana gráfica (o imprime en consola si no se dispone de pantalla gráfica):
```
Comment (tricantivu) [inline pages.es/common/lstopo.md:6]:
```suggestion
- Muestra la topología resumida del sistema en una ventana gráfica (o imprime en consola si no se dispone de una sesión gráfica):
```

File: pages.es/common/lstopo.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+`lstopo`
+
+- Muestra la topología completa del sistema sin resúmenes:
+
+`lstopo --no-factorize`
+
+- Muestra la topología resumida del sistema sólo con índices [físicos] (es decir, tal y como la ve el sistema operativo):
```
Comment (tricantivu) [inline pages.es/common/lstopo.md:14]:
```suggestion
- Muestra la topología resumida del sistema sólo con índices físicos (es decir, tal y como la ve el sistema operativo):
```

At the top of my head, I cannot think of a word suitable for a short option mnemonic. So, I opt for not having it all, what do you think @kant?

File: pages.es/common/lstopo.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+`lstopo --physical`
+
+- Escribe la topología completa del sistema en un archivo con el formato especificado:
+
+`lstopo --no-factorize --output-format {{console|ascii|tex|fig|svg|pdf|ps|png|xml}} {{ruta/al/archivo}}`
+
+- Salida en monocromo o escala de grises:
```
Comment (tricantivu) [inline pages.es/common/lstopo.md:22]:
```suggestion
- Muestra datos en monocromo o escala de grises:
```

File: pages.es/common/lstopo.md (hunk 2 above)
Comment (kant) [reply pages.es/common/lstopo.md:14]: @tricantivu I tried the best, but I agree in a whole with your point of view

**Review by tricantivu — APPROVED** [review]

**Review by spageektti — APPROVED** [review]
LGTM, thanks for your contribution.

**Pull request merged**